docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,557 @@
+deprecation on one release, and then completely remove them on the next one. This involves
+deleting the build scripts, the CI workflows, some metadata in `setup.cfg` and possibly some
+utility functions [^briefcase-utilities-example] and code paths [^briefcase-workaround-example]
+in napari itself.
+
+## Related Work
+
+napari has been using Briefcase to build its installers so far, as discussed in the "Motivation and
+scope" section. Other alternatives we considered before choosing `conda` were:
+
+* Adapting Briefcase to use `conda` packages, but it felt like we were rewriting `constructor` at
+  that point, only to get a barely working prototype.
+* Freezing `napari` directly with PyOxidizer [^pyoxidizer] and Nuitka [^nuitka]. These experiments
+  were not successful either and didn't allow for a good plugin installation story (the frozen
+  executables are immutable).
+
+## Implementation
+
+This NAP has described the whole strategy to implement a successful and comprehensive conda
+packaging story for napari. This work involves many moving pieces across different projects and
+tools, hence why a single PR is out of the question. In the following sections, we wll list the
+relevant PRs opened so far. Before that, though, we will propose a general strategy on how this
+infrastructure will be maintained and governed.
+
+### General strategy
+
+After many months operating on the `napari/napari` repository, we would like to propose the creation
+of a new repository (e.g. `napari/packaging`), where all these efforts can take place without
+getting in the way of the napari development itself.
+
+Most packaging tasks do not (and should not) rely tightly on the software being packaged,
+provided it follows the best practices. As such, the existing pipelines are mostly independent
+and could already live outside of `napari/napari`. Actually, it could be argued that they
+_should_ live outside to guarantee that the pipelines are not too tightly coupled.
+
+Given that Milestone 4 could benefit from having a specific tool to handle updates and manage
+existing installations (more details below), this repository could initially host its initial
+prototype, which would be designed in a napari-agnostic way for easy reusability in other
+projects facing our `constructor` improvements already [^mne-constructor].
+
+
+### Milestone 1: Distributing napari and plugins on conda-forge
+
+This work can be divided in two different tasks: adjusting the conda-forge feedstock for napari,
+and then migrating all the plugins over to conda-forge.
+
+* [Adjust outputs for multiple qt backends and menuinst=2](https://github.com/conda-forge/napari-feedstock/pull/32)
+* [Plugin migration search query](https://github.com/conda-forge/staged-recipes/pulls?q=is%3Apr+author%3Agoanpeca+created%3A2022-01-01..2022-05-01)
+  (~200 PRs)
+
+At the time of writing, the plugin migration to conda-forge can be considered 90% done, but a long
```

Comment:
something that has come up often: what happens to plugin developers that are "late to the party"?  Can they ping someone to get their plugin on conda-forge for them?  This concern has generally been met with "it's not that hard to get something on conda-forge" ... but specifically given the curated/moderated nature of conda-forge, I think we need to address that head on and state whether someone is going to be creating feedstocks for plugin devs on request, or whether everyone is on their own to deal with this process that definitely more time-consuming than a pypi deploy

Reply:
> Can they ping someone to get their plugin on conda-forge for them?

> whether someone is going to be creating feedstocks for plugin devs on request

If we can bring one or more Developer Advocates to the napari hub team, I think this would be a key responsibility they could take on.